link/kprobe.go

```diff
@@ -441,13 +449,9 @@ func createTraceFSProbeEvent(typ probeType, args probeArgs) error {
 		// r:ebpf_1234/r_my_kretprobe nf_conntrack_destroy
 		// p:ebpf_5678/p_my_kprobe __x64_sys_execve
 		//
-		// Leaving the kretprobe's MAXACTIVE set to 0 (or absent) will make the
-		// kernel default to NR_CPUS. This is desired in most eBPF cases since
-		// subsampling or rate limiting logic can be more accurately implemented in
-		// the eBPF program itself.
 		// See Documentation/kprobes.txt for more details.
 		token = kprobeToken(args)
```

Comment:
createTraceFSProbeEvent should return an error if maxactive is != 0 for anything but a kretprobe.